bucket/neohtop.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "1.1.1",
+    "description": "htop on steroids",
+    "homepage": "https://github.com/Abdenasser/neohtop",
+    "license": "MIT",
+    "url": "https://github.com/Abdenasser/neohtop/releases/download/v1.1.1/NeoHtop_1.1.1_x64.exe#/NeoHtop.exe",
+    "hash": "75f6ea91670bb4888bcb82572967c394570c5cfa7000fc30ea08a16981dfa77a",
+    "bin": "NeoHtop.exe",
+    "shortcuts": [
+        [
+            "NeoHtop.exe",
+            "NeoHtop"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/Abdenasser/neohtop",
```

Comment:
In this case, it lookslike the checkver can be simplified to 

```json
    "checkver": "github",
```

This is because:-
1. Regex is not needed as it matches the default regex
2. The URL provided to the GitHub parameter matches the project's url

Reply:
fixed in https://github.com/ScoopInstaller/Extras/pull/14478/commits/a1d26090b740ca62b226795d38b7f35b7428769e